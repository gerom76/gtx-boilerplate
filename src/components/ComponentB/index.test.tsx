import TestRenderer, { act } from 'react-test-renderer';
import { ComponentB } from '.';

// REMARKS: How to update snapshot for CI:
// yarn run test ComponentB -u
describe('ComponentB tests', () => {
  it('renders correctly', async () => {
    // act
    const sut = <ComponentB parameter1="MUI3" />;

    act(() => {
      const testRenderer = TestRenderer.create(sut);
      // assert
      const tree = testRenderer.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
