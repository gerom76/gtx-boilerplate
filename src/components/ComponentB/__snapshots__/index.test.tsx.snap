// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentB tests renders correctly 1`] = `null`;
